fastembed/embedding.py

```diff
@@ -401,6 +447,8 @@ class FlagEmbedding(Embedding):
         Embedding (_type_): _description_
     """
 
+    pooling_strategy: Literal["first_token", "mean"] = "first_token"
```

Comment:
`FlagEmbedding` class is not the parent class for `JinaEmbedding`. 

Flag is the specific implementation. Please do not add pooling to this and to `JinaEmbedding` directly?